grakn-client/src/main/java/ai/grakn/client/Client.java

```diff
@@ -18,103 +18,24 @@
 
 import javax.ws.rs.core.UriBuilder;
-import java.io.BufferedReader;
-import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
 import java.net.ProtocolException;
 import java.net.URL;
-import java.nio.charset.StandardCharsets;
-import java.util.Properties;
-
-import static ai.grakn.util.REST.Request.ID_PARAMETER;
-import static java.util.stream.Collectors.joining;
 
 /**
  * Providing useful methods for the user of the GraknEngine client
  *
  * @author alexandraorth
  */
-public class Client {
-
-    private static final Logger LOG = LoggerFactory.getLogger(Client.class);
-
-    private enum EngineStatus {
-        Running(0),
-        NotRunning(1),
-        Error(2);
-
-        private final int exitCode;
-
-        EngineStatus(int exitCode) {
-            this.exitCode = exitCode;
-        }
-    }
-
-    final ResponseHandler<Json> asJsonHandler = response -> {
-        try(BufferedReader reader = new BufferedReader(
-                new InputStreamReader(response.getEntity().getContent(), StandardCharsets.UTF_8))){
-            return Json.read(reader.lines().collect(joining("\n")));
-        }
-    };
-
-    final ResponseHandler<String> asStringHandler = response -> {
-        try(BufferedReader reader = new BufferedReader(
-                new InputStreamReader(response.getEntity().getContent(), StandardCharsets.UTF_8))){
-            return reader.lines().collect(joining("\n"));
-        }
-    };
-
-    public static void main(String[] args) {
-        EngineStatus engineStatus;
-        try {
-            engineStatus = checkServerRunning();
-        } catch (Exception e) {
-            LOG.error("An unexpected error occurred", e);
-            engineStatus = EngineStatus.Error;
-        }
-        System.exit(engineStatus.exitCode);
-    }
-
-    private static EngineStatus checkServerRunning() throws IOException {
-        String confPath = GraknSystemProperty.CONFIGURATION_FILE.value();
-
-        if (confPath == null) {
-            String msg = "System property `" + GraknSystemProperty.CONFIGURATION_FILE.key() + "` has not been set";
-            LOG.error(msg);
-            return EngineStatus.Error;
-        }
-
-        Properties properties = new Properties();
-        try (FileInputStream stream = new FileInputStream(confPath)) {
-            properties.load(stream);
-        }
-
-        String host = properties.getProperty(GraknConfigKey.SERVER_HOST_NAME.name());
-        int port = Integer.parseInt(properties.getProperty(GraknConfigKey.SERVER_PORT.name()));
-        if (serverIsRunning(new SimpleURI(host, port))) {
-            LOG.info("Server " + host + ":" + port + " is running");
-            return EngineStatus.Running;
-        } else {
-            LOG.info("Server " + host + ":" + port + " is not running");
-            return EngineStatus.NotRunning;
-        }
-    }
```

Comment:
For some reason most of this stuff has not been removed in the final diff :(

Reply:
@aelred wait more of this needs to die?

Reply:
If you look at the changes from all commits, `Client#main` is still there for example. But I removed it in my PR...